<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>199</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>110</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <column>
       <property name="text">
        <string>Город</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Местное время</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Опции</string>
    </property>
    <addaction name="addAlarm"/>
    <addaction name="listOfReminders"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="listOfReminders">
   <property name="text">
    <string>Список созданных напоминаний</string>
   </property>
  </action>
  <action name="addAlarm">
   <property name="text">
    <string>Создать напоминание</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
